stock_product_qty_by_packaging/models/stock_move_line.py

```diff
@@ -10,4 +10,4 @@ class StockMoveLine(models.Model):
     _name = "stock.move.line"
     _inherit = ["stock.move.line", "product.qty_by_packaging.mixin"]
 
-    _qty_by_pkg__qty_field_name = "reserved_qty"
+    _qty_by_pkg__qty_field_name = "quantity_product_uom"
```

Comment:
Mapping of `stock.move.line` fields before/after 17.0 from what I see:

- `reserved_qty` => `quantity_product_uom`
- `reserved_uom_qty` => `quantity`